src/styles/globals.css

```diff
@@ -70,7 +70,9 @@
     th {
         @apply tw-align-middle tw-border-b tw-border-solid tw-border-b-gray-200;
     }
-
+    tr:nth-child(even) {
+        @apply tw-bg-gray-100;
+    }
```

Comment:
To achieve the alternate table row shading effect. I think this is a great style to have in all our tables, that's why I left it as a global style. But let me know what you think!

![image](https://user-images.githubusercontent.com/59381432/190917511-2d0bafe8-8055-43fb-bc42-f20676abb8a2.png)

Reply:
Cool, I like it!